docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '3'
+services:
+  postgres:
+    image: "postgres"
+    container_name: ${COFFEE_DB_CONTAINER_NAME}
+    environment:
+      POSTGRES_USER: ${COFFEE_DB_USER}
+      POSTGRES_PASSWORD: ${COFFEE_DB_PASSWORD}
+      POSTGRES_DB: ${COFFEE_DB_NAME}
+      PG_DATA: /var/lib/postgresql/data/pgdata
+    ports:
+      - "${COFFEE_DB_PORT_NUMBER}:5432"
+    volumes:
+      - ./init_db.sh:/docker-entrypoint-initdb.d/init_db.sh
+    restart: always
+    healthcheck:
+      test: pg_isready -U ${COFFEE_DB_USER} -d ${COFFEE_DB_NAME}
+      interval: 10s
+      timeout: 5s
+      retries: 3
+    deploy:
+      resources:
+        limits:
+          cpus: '1'
+          memory: 4G
+
+  random_coffee:
+    image: "random_coffee:0.0.1"
```

Comment:
If the version changes, should we change it every time?

Reply:
We will always get version 0.0.1 in case of local using image

Reply:
ok